AequilibraEMenu.py

```diff
@@ -327,10 +327,7 @@ def run_log(self):
 
     def unload(self):
         del self.dock
-        try:
-            QgsApplication.processingRegistry().removeProvider(self.provider)
-        except:
-            pass
+        QgsApplication.processingRegistry().removeProvider(self.provider)
```

Comment:
What happens if the provider does not exist?

Reply:
Maybe something like this:

```suggestion
         if  self.provider in QgsApplication.processingRegistry().providers():
             QgsApplication.processingRegistry().removeProvider(self.provider)
```